### PR TITLE
Document multi component support

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: doc/multi-components.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.. code-block:: none
+
+    cabal haddock --haddock-html \
+                  --haddock-quickjump \
+                  --haddock-option="--use-index=../doc-index.html" \
+                  --haddock-option="--use-contents=../index.html" \
+                  --haddock-option="--base-url=.." \
+                  all
+
+The new `--base-url` option will allow to access the static files from the
```
Comment (bgamari) [inline doc/multi-components.rst:16]: I believe that should rather be
```
``--base-url``
```

Similarly below.

**Review by bgamari — CHANGES_REQUESTED** [review]

File: doc/multi-components.rst (hunk 1 above)
Comment (coot) [reply doc/multi-components.rst:16]: You're right, it's not markdown :).

Comment (coot) [reply doc/multi-components.rst]: Fixed.

**Comment by Mikolaj** [comment]
hi there! how is it going? I'm one of the very interested parties, both personally and as cabal maintainer. May I help perchance?

**Comment by Kleidukos** [comment]
@Mikolaj Thanks for the ping!
@bgamari Are you satisfied with the changes?
@coot Sorry for the latency >_>

**Review by bgamari — APPROVED** [review]

**Comment by Kleidukos** [comment]
@coot One conflict to solve and I'll merge the PR :)

**Review by Kleidukos — APPROVED** [review]

**Comment by coot** [comment]
@Kleidukos thanks

**Pull request merged**